src/main/java/com/iridium/iridiumskyblock/managers/TableManager.java

```diff
@@ -0,0 +1,110 @@
+package com.iridium.iridiumskyblock.managers;
+
+import com.j256.ormlite.dao.Dao;
+import com.j256.ormlite.dao.DaoManager;
+import com.j256.ormlite.support.ConnectionSource;
+import com.j256.ormlite.support.DatabaseConnection;
+import com.j256.ormlite.table.TableUtils;
+
+import java.sql.SQLException;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * @param <T> The Table Class
+ * @param <S> The Table Primary Id Class
+ */
+public class TableManager<T, S> {
+    private final List<T> list;
+    private final Dao<T, S> dao;
+
+    private final boolean autoCommit;
+    private final ConnectionSource connectionSource;
+
+    public TableManager(ConnectionSource connectionSource, Class<T> clazz, boolean autoCommit) throws SQLException {
+        this.connectionSource = connectionSource;
+        this.autoCommit = autoCommit;
+        TableUtils.createTableIfNotExists(connectionSource, clazz);
+        this.dao = DaoManager.createDao(connectionSource, clazz);
+        this.dao.setAutoCommit(getDatabaseConnection(), autoCommit);
+        this.list = dao.queryForAll();
+    }
+
+    /**
+     * Saves everything to the Database
+     */
+    public void save() {
+        try {
+            for (T t : list) {
```

Comment:
list.forEach(dao::createOrUpdate) is way cooler

Reply:
k but u wrong tho.

Cuz then if I do that the try/catch wont work since dao::createOrUpdate is its own function so it will be more like 
```
            list.forEach(t -> {
                try {
                    dao.createOrUpdate(t);
                } catch (SQLException throwables) {
                    throwables.printStackTrace();
                }
            });
``` 
which is bad and dum